modules/rhel-compute-requirements.adoc

```diff
@@ -23,7 +23,11 @@ ifdef::openshift-enterprise,openshift-webscale[]
 +
 [IMPORTANT]
 ====
-Only RHEL 7.8 - 7.9 is supported in {product-title} {product-version}. You must not upgrade your compute machines to RHEL 8.
+Adding RHEL 7 compute machines to an {product-title} cluster is deprecated. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+
+In addition, you must not upgrade your compute machines to RHEL 8 because support is not available in this release.
+
+For the most recent list of major functionality that has been deprecated or removed within {product-title}, refer to the _Deprecated and removed features_ section of the {product-title} release notes.
```

Comment:
As @katherinedube pointed out in https://github.com/openshift/openshift-docs/pull/32070#discussion_r623394061, we announced deprecation of this in 4.6, so this is updating the docs to reflect. 

@katherinedube and @miabbott , @marrusl - please scrutinize line 28 ("In addition, you must not upgrade...") to be sure this makes sense here. Thanks!

Reply:
lgtm. Thanks!